.circleci/config.yml

```diff
@@ -122,13 +122,15 @@ jobs:
             npm i cypress-downloadfile
             mkdir ~/project/cypress/pages
             node node_modules/.bin/cypress run --spec "cypress/integration/happy-path.spec.js"
-      # - run: 
-      #     name: Install and run webhint
-      #     command: | 
-      #       npm install --no-save hint
-      #       node node_modules/hint/dist/src/bin/hint.js https://$RELEASE_HOST/accessibility-statement
-      # - store_artifacts:
-      #     path: ~/project/hint-report/
+      - run: 
+          name: Install and run webhint
+          command: | 
+            npm install --no-save hint
+            cd ./cypress/pages/
+            node ../../node_modules/hint/dist/src/bin/hint.js ./
+            mv hint-report ../..
+      - store_artifacts:
+          path: ~/project/hint-report/
```

Comment:
Is there a reason that leaving the report where it is and specifying its path in `store_artifacts` doesn't work? That way strikes me as neater

Reply:
Ah ok no there are no issues with that. I did it this way simply because I thought this was neater 😂 as well as the fact that the hint-report is not associated with cypress and wanting to leave the `store_artifacts` the same

Reply:
OK. I don't see the rationale for not wanting to change the `store_artifacts` path but having it end up not in the cypress files makes some sense to me. I had expected there to be config to specify the destination for the report but I can't find any so moving it afterwards is fine.
Do we have to run webhint from where we do? Could we instead cd to where we want the report to be (`~/project/`) and run it there, with a path to the pages instead of `.`?

Reply:
I added the reason why in the description of one of my commits. I'll see if there is anyway way around it. Here's what I wrote:

_Webhint has a special command (./) used at the end for scanning an entire folder. When I tried added further folders to the command e.g. ./cypress/pages, it would error out.

The command seems be specific and only work when your in the folder to be scanned._

Reply:
Sorry, didn't see this until just now. I would wager you just might be in the wrong place for the path you used, to be honest

Reply:
Turns out you was very right! (as per usual)